Detectors/TPC/base/include/TPCBase/Sector.h

```diff
@@ -26,17 +26,15 @@ namespace TPC {
 class Sector
 {
   public:
-    enum
-    {
-        MaxSector = 36
-    };
+    // the number of sectors
+    static constexpr char MAXSECTOR=36;
```

Comment:
Indeed I would simply use an int / int64, since in any case the compiler will most likely optimise it to a `mov immediate`.